### PR TITLE
Removing one snapshot in down track.

### PR DESCRIPTION
**Pull request by boks1971**
Profiling showed updating jitter going through the snapshot maps. With the reduction of one, there should only be one snapshot and hopefully that should gain some cycles back.

### CODE REVIEW

File: pkg/sfu/downtrack.go [hunk 1]
```diff
@@ -762,16 +762,8 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int, paddingOnMute bool, forceMa
...
-	// LK-TODO Look at load balancing a la sfu.Receiver to spread across available CPUs
 	bytesSent := 0
 	for i := 0; i < len(snts); i++ {
-		// LK-TODO-START
-		// Hold sending padding packets till first RTCP-RR is received for this RTP stream.
-		// That is definitive proof that the remote side knows about this RTP stream.
-		// The packet count check at the beginning of this function gates sending padding
-		// on as yet unstarted streams which is a reasonable check.
-		// LK-TODO-END
-
```
Comment (boks1971) [inline pkg/sfu/downtrack.go:774]: some drive-by clean up/fix up. We have the information about receiver reports in `RTPStats` module. So, using that to check whether to send padding.

**Review by davidzhao — APPROVED** [review]
